docs/modules/mkdirp.md

```diff
@@ -0,0 +1,19 @@
+# mkdirp
+
+`mkdirp` can often be replaced with built-in Node APIs.

```

Comment:
does `mkdirp` _only_ export `mkdir` and `mkdirSync`? i.e. it has no other exported functionality/utils?

if so, we should probably add it to the native manifest and specify the correct node version there

Reply:
Modulo a couple minor differences (you can pass in a custom `fs` implementation to `mkdirp`).

As I mentioned above:
> This could be put in `native` if there's some way to attach documentation--I feel like at the moment, `native` can't really be used for anything more complex than polyfills where all you have to do is swap out the name of the function.

I see the `native` list as only for polyfills where the function behaves identically to the built-in one. If you need to pass in an extra argument, as is the case here, there should be some way to document that requirement.

Also, functions in the `native` manifest need to have an MDN path, which this does not, as it's not a JavaScript builtin but instead part of Node's standard library.

Reply:
ah sorry i missed that!

lets put it in `native` but keep the doc, and we can figure out how to provide doc links some other time

Reply:
actually, completely ignore me 😆 

sorry! lets keep it here as im really not sure how we should best handle node-specific natives yet